### PR TITLE
T3

### PR DESCRIPTION
**Pull request by ccccc170**
this pull request includes completed ticket 3 plus error handling for invalid path request

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            expect(!Array.isArray(topic)).toBe(true);
+            // tests that each object has two key/value pairs
+            expect(Object.keys(topic).length).toBe(2);
+            // tests that each object has desired keys and value input types
+            expect(topic).toEqual(
+              expect.objectContaining({
+                description: expect.any(String),
+                slug: expect.any(String),
+              })
+            );
```
Comment (HeatherEver) [inline __tests__/app.test.js:45]: we don't really need all these comments as the code itself is quite descriptive of what it's testing for

File: controllers/topics.controllers.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const { selectTopics } = require("../models/topics.models");
+
+exports.getTopics = (req, res, next) => {
+  selectTopics()
+    .then((topics) => {
+      res.send(topics);
```
Comment (HeatherEver) [inline controllers/topics.controllers.js]: don't forget to set the status here too

**Review by HeatherEver — APPROVED** [review]
Good work, just a couple of changes to make before merging

**Comment by ccccc170** [comment]
I have closed this pull request as I have implemnted the changes suggested by Heather and will open a new one after that commit.

**Pull request closed without merging**